src/web/server.rs

```diff
@@ -588,7 +588,9 @@ mod tests {
     #[actix_rt::test]
     async fn test_web_comment_plain_bad_chars() {
         let bad_bytes = &[142, 137, 138, 136, 140, 141, 10];
+        #[allow(clippy::string_lit_as_bytes)]
```

Comment:
nit: b"..." should be ok + adding bad_bytes: &[u8]

Reply:
I can `concat` the two arrays because they have different size. How should I address that?

Reply:
I just tryed:
```rust
let bad_bytes: &[u8] = &[142, 137, 138, 136, 140, 141, 10];
let input_vec = [b"/*char*/s: ", bad_bytes].concat();
```
and it's ok

Reply:
Oh, I forgot the & :sweat_smile: